Lecture8.tex

```diff
@@ -521,7 +521,7 @@ \subsection{Q-functions and their Evaluation}\label{ss:Q_eval}
 Similarly, the SARSA($\la$) algorithm uses
 \begin{align*}
 Q(s, a) & := Q(s,a) + \al_n(s,a) \cdot d_n e_n(s,a) \\
-\quad e_n(s,a)& := \ga \la e_{n-1}(s,a)+1\{s_n=1,a_n=a\} \,.
+\quad e_n(s,a)& := \ga \la e_{n-1}(s,a)+1\{s_n=s,a_n=a\} \,.
```

Comment:
In the indicator, we want the current state to be s, and not "1" (just some typo).